forge-gui/res/cardsfolder/m/moorland_rescuer.txt

```diff
@@ -3,7 +3,7 @@ ManaCost:5 W
 Types:Creature Human Knight
 PT:4/4
 T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigReturn | TriggerDescription$ When CARDNAME dies, return any number of other creature cards with total power X or less from the graveyard to the battlefield, where X is CARDNAME's power. Exile CARDNAME.
-SVar:TrigReturn:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | TargetMin$ 0 | TargetMax$ X | ValidTgts$ Creature.YouOwn+Other | MaxTotalTargetPower$ Y | SubAbility$ DBExile | TgtPrompt$ Select any number of other creature cards with total power X or less
+SVar:TrigReturn:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | ChangeType$ Creature.YouOwn+Other | WithTotalPower$ Y | ChangeNum$ X | Hidden$ True | AnyNumber$ True | SubAbility$ DBExile
```

Comment:
Might be nice to have a `SelectPrompt` or whatever the param is for ChangeZone (I can never remember)